### ISSUE TITLE
Angular error using in iconpicker in Umbraco 10

### ISSUE THREAD

**Issue by huwred** [denta/BG-Icon-Picker#53]
I have recently upgraded my Umbraco project to the laatest 10 version and the icon picker now throws the error below when opening a node that includes it and can no longer select an icon, page just freezes when you click the Add icon link.
It was working in Umbraco 9 so not sure what's changed.

`angular.js:15697 
        
       TypeError: Cannot read properties of null (reading 'classList')
    at Object.<anonymous> (umbraco.directives.min.js?d=637995163400000000:1:400753)
    at nouislider.min.js?umb__rnd=5e3c221e7dbd0715150a476c75a1f5f981de644b:1:21100
    at Array.forEach (<anonymous>)
    at nouislider.min.js?umb__rnd=5e3c221e7dbd0715150a476c75a1f5f981de644b:1:21078
    at Array.forEach (<anonymous>)
    at I (nouislider.min.js?umb__rnd=5e3c221e7dbd0715150a476c75a1f5f981de644b:1:21024)
    at nouislider.min.js?umb__rnd=5e3c221e7dbd0715150a476c75a1f5f981de644b:1:20750
    at Array.forEach (<anonymous>)
    at Object.X [as on] (nouislider.min.js?umb__rnd=5e3c221e7dbd0715150a476c75a1f5f981de644b:1:20728)
    at setUpActivePipsHandling (umbraco.directives.min.js?d=637995163400000000:1:400472)`

**Comment by bgunnarsson** [comment]
Yeah I'd have to create specific versions for 9+ - I've been contemplating if I should do it or not since there isn't much activity here anymore.
